integrated_channels/integrated_channel/transmitters/learner_data.py

```diff
@@ -115,56 +124,65 @@ def assessment_level_transmit(self, exporter, **kwargs):
             TransmissionAudit=TransmissionAudit,
         )
 
-        # Retrieve learner data for each existing enterprise enrollment under the enterprise customer
-        # and transmit the data according to the current enterprise configuration.
-        for learner_data in exporter.bulk_assessment_level_export():
-            serialized_payload = learner_data.serialize(enterprise_configuration=self.enterprise_configuration)
-            enterprise_enrollment_id = learner_data.enterprise_course_enrollment_id
-            lms_user_id = LearnerExporterUtility.lms_user_id_for_ent_course_enrollment_id(
-                enterprise_enrollment_id)
-
-            # Check the last transmission for the current enrollment and see if the old grades match the new ones
-            if is_already_transmitted(
-                    TransmissionAudit,
-                    enterprise_enrollment_id,
-                    learner_data.grade,
-                    learner_data.subsection_id
-            ):
-                # We've already sent a completion status for this enrollment
-                LOGGER.info(generate_formatted_log(
-                    app_label, enterprise_customer_uuid, lms_user_id, learner_data.course_id,
-                    'Skipping previously sent EnterpriseCourseEnrollment {}'.format(enterprise_enrollment_id)
-                ))
-                continue
-
-            try:
-                code, body = self.client.create_assessment_reporting(
-                    getattr(learner_data, kwargs.get('remote_user_id')),
-                    serialized_payload
-                )
-                LOGGER.info(generate_formatted_log(
-                    app_label, enterprise_customer_uuid, lms_user_id, learner_data.course_id,
-                    'create_assessment_reporting successfully completed for EnterpriseCourseEnrollment {}'.format(
+        if not self.enterprise_configuration.disable_learner_data_transmissions:
```

Comment:
similarly here, perhaps it would be more clear to use an `if` block that concisely logs + returns early before the meat of the method